tests/functional/a/assigning/assigning_non_slot.py

```diff
@@ -147,7 +147,7 @@ class ClassReassingingInvalidLayoutClass:
     __slots__ = []
 
     def release(self):
-        self.__class__ = ClassWithSlots # [assigning-non-slot]
+        self.__class__ = ClassWithSlots
```

Comment:
This feels incorrect upon second review. This is an instance attribute and should still flag right?